src-tauri/src/mod_manager.rs

```diff
@@ -35,23 +38,70 @@ pub fn deploy(mods: Vec<Mod>, game: Game) {
 pub fn scan_games() -> Vec<String> {
   let mut steam_games: Vec<String> = Vec::new();
   let steam_apps = SteamDir::locate().unwrap().apps().clone();
+
+  let mut known_path_extensions_json = dirs::config_dir().unwrap().join("tmm_stage").join("known_path_extensions");
```

Comment:
Crashes on scan games : No such file or directory of this type.

Reply:
I mentioned this in the README.md. I'm currently storing the known_path_extensions.json and the supported_games.json in the config dir and not handling what happens if they don't exist. I plan on moving them to the tauri dist folder, so they're bundled with the application when building. Since this is my first application with tauri though, I will still have to learn how to do that.